kingpin/actors/aws/ecs.py

```diff
@@ -488,7 +499,8 @@ def __init__(self, *args, **kwargs):
         super(RunTask, self).__init__(*args, **kwargs)
         self.task_definition = self._load_task_definition(
```

Comment:
This looks great, but lets also apply this to the `self.service_definition` too for consistency.